Playbooks/Block-AADUser/alert-trigger/azuredeploy.json

```diff
@@ -4,17 +4,18 @@
     "metadata": {
         "title": "Block AAD user - Alert",
         "description": "For each account entity included in the alert, this playbook will disable the user in Azure Active Directoy, add a comment to the incident that contains this alert and notify manager if available. Note: This playbook will not disable admin user!",
-        "prerequisites": "",
+        "prerequisites": [""],
         "postDeployment": ["1. Assign Microsoft Sentinel Responder role to the Playbook's managed identity.", "2. Grant User.Read.All, User.ReadWrite.All, Directory.Read.All, Directory.ReadWrite.All permissions to the managed identity.", "3. Authorize Azure AD and Office 365 Outlook Logic App connections."],
-        "lastUpdateTime": "2021-11-25T00:00:00.000Z",
+        "lastUpdateTime": "2022-07-11T00:00:00.000Z",
```

Comment:
@v-sabiraj : we should add this prop with default value as current date only if it does not exist, please revert this date change